apps/sptribs/sptribs-cron-migrate-case-flags/aat/00.yaml

```diff
@@ -6,7 +6,7 @@ spec:
   releaseName: sptribs-cron-migrate-case-flags
   values:
     job:
-      schedule: "0/10 * * * *"
+      schedule: "15,45 * * * *"
```

Comment:
Aren't we going to restrict this job to run in the morning?
15,45 8-12 * * *

Reply:
Updated to 8-16 for Case flags as the cron job is ready for Flags.